### ISSUE TITLE
questions about get_logits

### ISSUE THREAD

**Issue by wswaq** [ExplainableML/flair#2]
1. in get_logits,encode_image return 2 value ,but you use one param image_embeddings , and then you put it in visual_proj,is it a mistake?
2. your infer code and paper actually doesn't use the local text tokens ,but in get_logits,you use local ext_tokens  to get conditioning features ,is it a mistake?

**Comment by sean-xr** [comment]
Hi, thanks for the nice catch! the `get_logits()`in `model.py` is actually not used either in training or inference, so it should not affect the results. This function is indeed errornous, i removed it to avoid further confusion. Feel free to ask if you meet any problems.

**Comment by wswaq** [comment]
hello,i want to reproduce your methods,so i use the get_logits to find that how can i reproduce the loss? While you remove it , i wonder how can i train the model use flair? such as if i have image text pairs,what ' s the loss use flair?

**Comment by SmartNight-cc** [comment]
> hello,i want to reproduce your methods,so i use the get_logits to find that how can i reproduce the loss? While you remove it , i wonder how can i train the model use flair? such as if i have image text pairs,what ' s the loss use flair?

Hello! I have a similar problem.😔I can't find where exactly in the code is trained using local_feature. Have you solve it?

**Comment by sean-xr** [comment]
Hello guys, I'm sory for the late reply.
The training code is out and you can see how the loss function works in `src/flair/loss.py`. I will also update the README for detailed usage for training FLAIR. Thanks for the patience and please let me know if there are any further questions.
